cli/README.md

```diff
@@ -61,6 +61,17 @@ To process json file containing sequence of events :
 run `node cli.mjs --sequence "path of sequence.json"`
 example : node cli.mjs --sequence ../examples/device-onDeviceNameChanged.sequence.json
 
+## Status of ws connection
+
+To get the status of web-socket connection of user :
+run `node cli.mjs --getStatus`
+
+## Adding user
+
+To add user :
+run `node cli.mjs --addUser <userId>`
+example: node cli.mjs --addUser "123~A#netflix"
```

Comment:
Did we land where we support both POST and PUT? If so, do we have two separate CLI params/commands or just one with an optional userId value? If the former, we should document both. If the latter, we should probably put square brackets around userId and show two examples, one with a user id and one without (with a note that a UUID will be generated).